This branch is crashing with an error:
```
@__swiftmacro_6Client20RemoteTabsPanelState15CopyWithUpdatesfMm_.swift:9:9: error: argument 'windowUUID' must precede argument 'refreshState'
```

--- a/firefox-ios/Client/Frontend/Browser/Tabs/State/RemoteTabsPanelState.swift
+++ b/firefox-ios/Client/Frontend/Browser/Tabs/State/RemoteTabsPanelState.swift
@@ -51,11 +51,11 @@ enum RemoteTabsPanelEmptyStateReason {
 /// State for RemoteTabsPanel. WIP.
 @CopyWithUpdates
 struct RemoteTabsPanelState: ScreenState, Sendable {
+    let windowUUID: WindowUUID
     let refreshState: RemoteTabsPanelRefreshState
     let allowsRefresh: Bool
     let clientAndTabs: [ClientAndTabs]
     let showingEmptyState: RemoteTabsPanelEmptyStateReason?// If showing empty (or error) state
-    let windowUUID: WindowUUID
     let devices: [Device]
 
     init(appState: AppState, uuid: WindowUUID) {

